### ISSUE TITLE
DPL-742 - As developers we need to modify pacbio plates so that they can be distinguished for a Revio run

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-service#991]
**User story**
As developers we need to modify pacbio plates so that they can be distinguished for a Revio run.

**Who are the primary contacts for this story**
Stev / Carol / James / Craig

**Who is the nominated tester for UAT**
PSD

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] A pacbio plate needs a sequencing kit box barcode
- [x] A pacbio plate needs a number
- [x] All plates must have a sequencing kit box barcode and plate nunber
- [x] add a migration to add a plate number and sequencing kit box barcode to all existing plates
- [x] Make run sequencing kit box barcode optional
- [x] Remove references to adding sequencing kit box barcode to run
- [x] Ensure sample sheet uses plate sequencing kit box barcode
- [ ] After production migration remove sequencing kit box barcode from run (roll into another story).

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by harrietc52** [comment]
## Deployment

See comment [here](https://github.com/sanger/traction-service/issues/976#issuecomment-1607690429)
